docs/releases/1.11.29.1.txt

```diff
@@ -0,0 +1,30 @@
+============================
+Django 1.11.29 release notes
```

Comment:
Both of these version numbers need corrected.

Reply:
Changed.